apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QSchemaUserTypeIT.java

```diff
@@ -292,27 +292,9 @@ public void udtDelete() {
     // and then verify it doesn't exist any more
     verifyTypeNotFound(testKeyspaceName(), typeName);
 
-    // Then try 2 invalid cases; first, trying to delete again
-    String response =
-        givenWithAuth()
-            .when()
-            .delete(deletePath)
-            .then()
-            .statusCode(HttpStatus.SC_BAD_REQUEST)
-            .extract()
-            .asString();
-    // 23-Aug-2022, tatu: Not optimal, straight gRPC error but it is what it is:
-    ApiError apiError = readJsonAs(response, ApiError.class);
-    assertThat(apiError.code()).isEqualTo(HttpStatus.SC_BAD_REQUEST);
-    assertThat(apiError.description())
-        // Further: C*3/C*4 have different failure messages
-        // C*3: "Invalid argument for gRPC operation (INVALID_ARGUMENT->Bad Request):
-        //   INVALID_ARGUMENT:
-        //   No user type named udt_ks_udtDelete_1662051039657.test_udt_delete exists."
-        // C*4: "Invalid argument for gRPC operation (INVALID_ARGUMENT->Bad Request):
-        //   INVALID_ARGUMENT:
-        //   Type 'udt_ks_udtDelete_1662052163437.test_udt_delete' doesn't exist"
-        .matches(String.format("Invalid argument.*%s.*exist.*", typeName));
+    // Then try 2 invalid cases; first, trying to delete again: deletion now
+    // idempotent (same as other resources) due to "ifExists", so
+    givenWithAuth().when().delete(deletePath).then().statusCode(HttpStatus.SC_NO_CONTENT);
```

Comment:
Since the behavior is to use "ifExists" -- same as with all other operations -- there will not be failure for retrying delete operation. This should have been changed earlier but for some reason hasn't been.